apps/public-docsite-v9/src/DocsComponents/ReactIconGrid.stories.tsx

```diff
@@ -0,0 +1,268 @@
+import * as ReactIcons from '@fluentui/react-icons';
+import {
+  Button,
+  Field,
+  Input,
+  Link,
+  makeStyles,
+  MessageBar,
+  Radio,
+  RadioGroup,
+  shorthands,
+  Toast,
+  Toaster,
+  ToastTitle,
+  tokens,
+  useId,
+  useIsomorphicLayoutEffect,
+  useScrollbarWidth,
+  useToastController,
+} from '@fluentui/react-components';
+import * as React from 'react';
+import { FixedSizeGrid, type GridChildComponentProps } from 'react-window';
```

Comment:
Just curious, don't think you need to change this, but why not use the [Fluent virtualizer](https://react.fluentui.dev/?path=/docs/preview-components-virtualizer--default)?

Reply:
@spmonahan this example is using grid layout. AFAIK Fluent virtualizer does not support it.